docs/api.rst

```diff
@@ -427,6 +427,66 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+Transfers
+=========
+
+Initiating a Transfer
+---------------------
+
+You can create a new transfer by making a ``PUSH`` to the following endpoint along with a json payload containing
```

Comment:
I also think we should mention that this request will only return once the transfer succeeded or failed, and that the failure is determined (currently) by the lock's expiration (we need to implement initiator timeouts)